physionet-django/console/views.py

```diff
@@ -2013,8 +1992,7 @@ def submission_stats(request):
                   {'stats_nav': True, 'submenu': 'submission', 'stats': stats})
 
 
-@login_required
-@user_passes_test(is_admin, redirect_field_name='project_home')
+@permission_required('user.can_view_access_logs', raise_exception=True)
```

Comment:
The `user.can_view_access_logs` permission doesn't seem to exist in the list of available user permissions. The handling editor is not able to view: http://localhost:8000/console/user-access-logs/

Reply:
Fixed: [3435cb0](https://github.com/MIT-LCP/physionet-build/pull/1487/commits/3435cb075d73af6452a404b1aa1c7dbf1dc2d628)